### PR TITLE
build wheels script bug fix

### PR DESCRIPTION
**Pull request by taylanbil**
script currently quits because it cannot find conda
somehow the `source ~/.bashrc` line is not changing the path within
script, so it quits when it cannot find conda.

apply_patches was executed using a relative path, made that
semi-absolute.

### CODE REVIEW

File: scripts/build_torch_wheels.sh [hunk 1]
```diff
@@ -67,7 +67,8 @@ function build_and_install_torch() {
   # Only checkout dependencies once PT commit ID checked out.
   git submodule update --init --recursive
   # Apply patches to PT which are required by the XLA support.
-  xla/scripts/apply_patches.sh
+  # xla/scripts/apply_patches.sh
```
Comment (jysohn23) [inline scripts/build_torch_wheels.sh]: Why did we comment this out?

Comment (dlibenzi) [reply scripts/build_torch_wheels.sh]: Cannot be commented out.

Comment (taylanbil) [reply scripts/build_torch_wheels.sh]: I could remove the commented line. script is executed in the next line.

Comment (taylanbil) [reply scripts/build_torch_wheels.sh]: to clarify; apply_patches.sh is executed in line 71. just not via a relative path.

File: scripts/build_torch_wheels.sh [hunk 2]
```diff
@@ -67,7 +67,8 @@ function build_and_install_torch() {
...
+  $(echo $0 | sed s/build_torch_wheels/apply_patches/)
```
Comment (taylanbil) [inline scripts/build_torch_wheels.sh]: @dlibenzi @jysohn23 this line is equivalent to line 70 when run from the pytorch directory, but it doesn't fail when run from somewhere else (which I incidentally did today).

File: scripts/build_torch_wheels.sh (hunk 1 above)
Comment (dlibenzi) [reply scripts/build_torch_wheels.sh]: Now that I see that ... why that weird sed replace?
Can't you just append the patch script name to a `dirname $0`?

Comment (taylanbil) [reply scripts/build_torch_wheels.sh]: sure, changing to dirname

Comment (taylanbil) [reply scripts/build_torch_wheels.sh]: and removing comment.

**Pull request merged**